03_gamedesign/index.tex

```diff
@@ -1,5 +1,75 @@
 \chapter{Game Design}
 
+\section{Design Approach}
+
+A game was designed to meet the requirements outlined by the specification to provide structure to the inter-island interactions and decisions. This \emph{game design framework} provides a basis for the simulations, experiments and evaluations into different agent strategies and system designs.
+
+The game is structured as series of \emph{seasons} and \emph{turns}, which concludes once all of the islands die. The objective for the islands is to survive for as many turns and Seasons as possible. The following definitions provide a detailed outline of the relevant terminology used to formalise the game definition and description.
+
+\begin{definition} \label{def:gamestart}
+    \textbf{Game start} is defined as the beginning of a simulation. Each simulation can be configured using a number of parameters, including but not limited to the cost of living, probability of a disaster and initial resources in the common pool.
+\end{definition}
+
+\begin{definition} \label{def:turn}
+    A \textbf{turn} is defined as a series of exchanges between agents in which islands receive resource updates, attend Inter-Island Organisation meetings, and interact with one another and the game state through \textbf{actions}. A turn can be broken down as follows:
+    \begin{enumerate}
+        \item Resource Updates for each island and on global game state.
+        \item IIGO decides rule changes, elections, sanctions.

```

Comment:
at this point in the report, 
IIGO, IITO, IIFO ... these are NOT defined I think :)

so.. you gotta define them :)
Inter-island ..... (IIGO) kinda thing should do the work

Reply:
good point can add